drivers/at86rf2xx/at86rf2xx_internal.c

```diff
@@ -205,7 +220,7 @@ void at86rf2xx_configure_phy(at86rf2xx_t *dev)
     at86rf2xx_set_state(dev, prev_state);
 }
 
-#if defined(MODULE_AT86RF233) || defined(MODULE_AT86RF231)
+#if defined(MODULE_AT86RF233) || defined(MODULE_AT86RF231) || defined(MODULE_AT86RFA1) || defined(MODULE_AT86RFR2)
 void at86rf2xx_get_random(const at86rf2xx_t *dev, uint8_t *data, size_t len)
```

Comment:
Totally unrelated to this PR, but I think that this is a horrible idea. The proper way of doing that would be to just pass the raw RSSI to some upper entropy collection code and give a hint of how many entropy bits we expect in each chunk of data passed to the upper entropy harvesting code. And the canonical way of transforming a sequence of *m* bits which contains *n* bits of entropy (with *n* << *m*) into a sequence of *n* bits containing (in an ideal world) *n* bits of entropy is to use a cryptographic hash function. I might just open a PR to do so.

Reply:
The [data sheet](http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-8266-MCU_Wireless-ATmega128RFA1_Datasheet.pdf#page=110) explicitly denotes those bits as random.

Just rest assured that currently there is no code inside RIOT that makes use of this function.
Although it sure would be useful to expose it over a common interface if `periph/hwrng` does not exist.